Refit.Tests/RestService.cs

```diff
@@ -73,15 +73,27 @@ public interface IApiBindPathToObject
 
         [Post("/foos/{request.someProperty}/bar/{request.someProperty2}")]
         Task PostFooBar(PathBoundObject request, [Body]object someObject);
+
+        [Get("/foos/{request.someProperty}/bar/{request.someProperty2}")]
+        Task GetFooBars(PathBoundObjectWithQuery request);
+
+        [Multipart]
+        [Post("/foos/{request.someProperty}/bar/{request.someProperty2}")]
+        Task<HttpResponseMessage> PostFooBarStreamPart(PathBoundObject request, StreamPart stream);
```

Comment:
Can you please add another variant of this where it has a signature like:

```
[Multipart]
        [Post("/foos/{request.someProperty}/bar/{request.someProperty2}")]
        Task<HttpResponseMessage> PostFooBarStreamPart(PathBoundObject request, [Query] ModelObject someQueryParams, StreamPart stream);
```

Reply:
That "should" put the `ModelObject` properties on the query string and the `StreamPart` should be the multipart content. If that's not happening, that'd be a bug.

Reply:
ok yeah it appears to be a bug in the existing code, should I try to fix in this PR?

Reply:
Please!

Reply:
Ok my latest commit has the test cases and the fix for this